src/components/multi-line-plot/MultiLinePlot.js

```diff
@@ -18,6 +18,7 @@ import {
   getXAxisDataKey,
   getYAxisDataKey,
   getYAxisTickFormatter,
+  sortBySeriesName,
 } from '../../utils/plotConfigGetters';
 import GeneralChartComponents from '../general-chart-components/GeneralChartComponents';
 import PlotContainer from '../plot-container/PlotContainer';
```

Comment:
We should capturei n a ticket the fact that we want to be able to sort this by a column's sorter (piece of info on a column).

TODO: Ensure a numerical sort works, like a numerical dimension, days since last order, ensure I test that, JS `.sort` might break

Reply:
No changes in plot files, just plotConfigGetters